tests/output/InteractionsTest.py

```diff
@@ -4,19 +4,36 @@
 
 from coalib.output.Interactions import fail_acquire_settings
 from coalib.output.printers.LogPrinter import LogPrinter
+from testfixtures import LogCapture
 from coalib.settings.Section import Section
 
 
 class InteractionsTest(unittest.TestCase):
 
     def test_(self):
         log_printer = LogPrinter(NullPrinter())
-        section = Section('')
-        self.assertRaises(TypeError, fail_acquire_settings, log_printer, None,
-                          section)
+        self.assertRaises(TypeError, fail_acquire_settings, log_printer,
+                          None)
+
         self.assertRaises(AssertionError,
                           fail_acquire_settings,
                           log_printer,
-                          {'setting': ['description', 'bear']}, section)
-        self.assertEqual(fail_acquire_settings(log_printer, {}, section), None,
-                         section)
```

Comment:
oh interesting, actually this was coded badly before, the `section` parameter here is for the `message` parameter of `assertEqual`. If you like, you can remove it with a separate commit to make your changes more atomic (would be quite good I think + you get one commit credited more :D)